### PR TITLE
Implement Pardon

### PR DESCRIPTION
**Pull request by calder**
Did I fuck this up?

### CODE REVIEW

File: src/Game/Cosanostra/Action.hs [hunk 1]
```diff
@@ -68,6 +68,7 @@ targetSelectors Follow{} = [alive]
 targetSelectors Investigate{} = [alive]
 targetSelectors RoleInvestigate{} = [alive]
 targetSelectors Autopsy{} = [dead]
+targetSelectors Pardon{} = [alive]
```
Comment (rfw) [inline src/Game/Cosanostra/Action.hs]: allow self-pardons?

File: src/Game/Cosanostra/Resolver/Action.hs [hunk 2]
```diff
@@ -438,6 +439,23 @@ appliesEffects RoleInvestigate{} _ _ =
 appliesEffects Autopsy{} _ _ =
     M.empty
 
+appliesEffects Pardon{} act _
+  | source == target = M.empty
+  | otherwise = M.fromList
+     [ (target, [Effect
```
Comment (rfw) [inline src/Game/Cosanostra/Resolver/Action.hs]: move this to line above, and just write `[(target, ...`

File: src/Game/Cosanostra/Action.hs (hunk 1 above)
Comment (calder) [inline src/Game/Cosanostra/Action.hs]: Fixed.

File: src/Game/Cosanostra/Resolver/Action.hs (hunk 2 above)
Comment (calder) [inline src/Game/Cosanostra/Resolver/Action.hs]: Done.

File: .gitignore [hunk 3]
```diff
@@ -7,6 +7,7 @@ cabal-dev
 *.chs.h
 *.dyn_o
 *.dyn_hi
+.DS_Store
```
Comment (rfw) [inline .gitignore]: lol

**Comment by rfw** [comment]
conflicts? :(

**Comment by rfw** [comment]
otherwise looks good

**Pull request merged**